telegram/bot.py

```diff
@@ -1985,14 +2008,24 @@ def answer_pre_checkout_query(self, pre_checkout_query_id, ok, error_message=Non
             :class:`telegram.TelegramError`
 
         """
-        url = '{0]/answerPreCheckoutQuery'.format(self.base_url)
+
+        if ((ok is not True and error_message is None) or
+                (ok is True and error_message is not None)):
+            raise TelegramError(
+                'answerPreCheckoutQuery: If ok is True, there should '
+                'not be error_message; if ok is False, error_message '
+                'should not be empty')
```

Comment:
okay for this one i believe i checked with the other existing methods beforehand about these mutual exclusive parameters
i raise exception there was because i was following `edit_message_caption` and similar ones with either inline messages or normal messages
what do you guys think?